easybuild/easyconfigs/e/ESyS/ESyS-2.3.5-foss-2016b-Python-2.7.12.eb

```diff
@@ -1,8 +1,10 @@
+# @author: jgp, gppezzi (CSCS)
```

Comment:
we can use `git blame` as a last resort, but it is nicer if people claim authorship. Besides, the latter option will also work if the file is moved to another git repo.